staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -641,11 +642,41 @@ func (meal *cfgMeal) imaginePL(proto *flowcontrol.PriorityLevelConfiguration, re
 	}
 }
 
+type immediateQueueSetCompleter struct{}
+
+func (immediateQueueSetCompleter) Complete(dCfg fq.DispatchingConfig) fq.QueueSet {
+	return immediateQueueSet{}
+}
+
+type immediateQueueSet struct{}
+
+func (immediateQueueSet) BeginConfigChange(qCfg fq.QueuingConfig) (fq.QueueSetCompleter, error) {
+	return immediateQueueSetCompleter{}, nil
+}
+
+func (immediateQueueSet) IsIdle() bool {
+	return false
```

Comment:
This is not necessarily a correct answer.  See https://github.com/tkashem/kubernetes/blob/panic/staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/interface.go#L64-L65